plugin.go

```diff
@@ -44,6 +44,9 @@ func (p Plugin) Exec() error {
 
 	var commands []*exec.Cmd
 	remote := p.Config.Remote
+	if len(p.Config.Secrets) != 0 {
+		exportSecrets(p.Config.Secrets)
```

Comment:
I do not think we need this.  If you add secrets with `drone secret add` then they are automatically injected as ENVVARS to the container.  So in your example cause you would `drone secret add AWS_ACCESS_KEY_ID` rather then add some some other secret value that you then assign to `AWS_ACCESS_KEY_ID` via this function.